### PR TITLE
The getCity method is created on all layers; unit tests and integrati…

### PR DESCRIPTION
**Pull request by ghilyana**
…on test are added.

### CODE REVIEW

File: src/main/java/com/ghilly/service/CityService.java [hunk 1]
```diff
@@ -4,4 +4,6 @@
 
 public interface CityService {
     City create(String cityName, int countryId);
+
+    City getCity(int cityId, int countryId);
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/service/CityService.java]: Why do you need countyId?

File: src/main/java/com/ghilly/web/controller/CityController.java [hunk 2]
```diff
@@ -26,4 +23,11 @@ public ResponseEntity<City> create(@RequestBody String cityName, @PathVariable i
         logger.info("The city with the name {} is created.", cityName);
         return ResponseEntity.ok().body(city);
     }
+
+    @GetMapping("/{cityId}")
+    public ResponseEntity<City> getCity(@PathVariable int cityId, @PathVariable int countryId) {
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/web/controller/CityController.java]: Why do you need coutryId?

File: src/main/java/com/ghilly/service/CityService.java (hunk 1 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/service/CityService.java]: Deleted

File: src/main/java/com/ghilly/web/controller/CityController.java (hunk 2 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/web/controller/CityController.java]: Deleted

**Review by gregorusanov — APPROVED** [review]

**Review by gregorusanov — APPROVED** [review]

**Pull request merged**